### ISSUE TITLE
Coordinate accessibility events/clinics/workshops for the community

### ISSUE THREAD

**Issue by isabela-pf** [Quansight-Labs/czi-scientific-python-mgmt#74]
Part of this project is to provide accessibility education events/clinics/workshops/some other popular synonym to bring accessibility knowledge to the community and document it accordingly. I’ve been tasked with coming up with at least two events for this. I have three proposed event topics. We may choose two or end up doing all three. My proposed topics (in no particular order) are

1.  Accessibility in your digital day-to-day. An event outlining accessibility considerations relating to common tasks to encourage people to better their everyday digital interactions with smaller actions that add up.
2. Recommendations for more accessible data visualizations. (A popular request.)
3. FAQ: Frequent Accessibility Quagmires. Documenting the sensory impacts of popular accessibility issues on the Internet, ways to test for them, and ways to improve them. The main draw of this over more general accessibility events would be that this event actually demonstrates what happens when things are built less accessibly as well the methods to discover them, more of an opportunity for interaction and understanding than being told a list of things to follow without thinking.

I have more details on format for these events, but I do want to check that these topics seem fitting before I dive too deep into ironing out each detail. They need to be completed by December.

I did not find an issue for this previously on this repository, but if there is one please let me know and I’ll consolidate information to it. Thank you!

cc: @trallard

**Comment by trallard** [comment]
Agreed topics:

1. Data vis 🎨 


Ideas & discussions
1. Accessibility in your digital day-to-day
  - People wanting to do better but perceiving a lack of deep a11y knowledge as a barrier 
3. FAQ
  - what happens when items are inaccessible?

**Comment by isabela-pf** [comment]
Summarizing the discussion: 

## Topics!
- Data visualization as a topic is totally agreed upon.
- Accessibility in your day to day is the next most likely. More examples/clarity is wanted.
- FAQ is a backup option. There are concerns about the work it will take to have so many demonstrations ready.
- Focus more on a scientific spin than totally general accessibility applications.

## Scheduling!
- Schedule two events 4 weeks apart from one another.
- Schedule Accessibility in your day to day first. Schedule Data visualization second.
- I am proposing the dates be **Tuesday, October 15** and **Tuesday, November 19**. I could also do **Tuesday, September 17** and **Tuesday, October 15** if we want to start a month earlier.

**Comment by isabela-pf** [comment]
I’m adding a rough agenda outline for the accessibility in digital day to day topic for clarification and feedback. Right now this is written for about 1.5 hours since I don’t want to plan anything too long it is hard to commit to or schedule, but I don’t want to lose all the time setting up.

## Concept

Taking action to try and improve access of the world around you to disabled people is not often a clear path. Some of the main feedback I’ve heard from people in advocating for accessibility over the years is that knowing it is important makes sense, but figuring out what to do is a challenge and taking action on limited resources—time, energy, and power—makes all of it much more difficult.

Even so, there’s so many opportunities in everyday routines online to complete tasks with more consideration of disabilities. The goal in this workshop is to give attendees examples of immediately actionable accessibility considerations in things they likely already do, to help them notice opportunities for accessibility, provide practice with well-scoped actions, and make a habit of individuals making online spaces more accessible. To do this, the format will be:
- Go from beginning to end of a day via a set of common online tasks (ie. writing an email, inviting people to a social event). 
- At each task, the audience will have at least two options of how to complete that task. They will choose an via Zoom poll. 
- Like a choose-your-own-adventure story, the option with the most votes will lead to a slide with the outcome. Choices that consider accessibility will be noted and a quick “why they help” will be covered on the slide. Choices that miss accessibility considerations will describe its negative impacts and point to what to do instead.
- A tally of choices will be mentioned at the end to help the group see how these small choices can add up by the end of a day. 

## Draft agenda outline

- Pre-workshop filing in and hellos.
- Introductions.
- Summarize agenda and give out links. Code of conduct.
- Explain the format and expectations.
     - What we’ll cover in this workshop
     - What we will not cover in this workshop
     - How we’ll play (scenario, options, vote, results)
     - Rules
- Trial run scenario (not accessibility related)
- Real scenario. Let’s start the day!
- Say good morning to loved ones
     - Happens in messaging software. Goal: remember this is about people, and people all benefit from different things. Know your audience.
- Check email and/or Slack
     - Happens in email software or messaging software. Can be in the browser or not. Goal: Use complete sentences, relevant grammar, and be thoughtful about acronyms and jargon.
- Review your calendar and schedule a meeting.
     - Happens in web-based calendar. Goal: Provide a meaningful meeting invite with relevant support. Give it a meaningful title. Provide an agenda and/or things attendees need to know before attending. Provide opportunity to reschedule and request something from the meeting host (ie. slides ahead of time).
- Present something at work.
     - Could happen virtually or in-person, but it is likely digital supports like a document or slides or a demonstration will be there too. Goal: Have a table of contents. Define terms. Briefly describe the contents and meaning of what you are presenting, especially diagrams or demonstrations. Use headings and appropriate formatting to section contents.
- Scenario lunch! Take a break in the scenario and in real life too.
- Break (5 minutes)
- Send something fun to your friends.
     - Happens in messaging software. Goal: Provide meaningful text for any links sent. Check if that cute animal video needs sound or has captions.
- Host a meeting.
     - Could happen virtually or in-person. Goal: Plan breaks, especially for anything longer than an hour. Choose a venue that will meet the needs of all your attendees. Minimize or eliminate side conversations (ie. audibly, in the meeting chat, elsewhere within the group). Consider providing captioning or interpreters; make this a requirement if planning something public facing. Set expectations for speaking order (ie. raise hands or other signal). Have shared group notes.
- Share a social invite. 
     - Happens via messaging software with the invite as an image. Goal: Share the invite with relevant information in text outside the image. This could be alt text or it could be in the message itself. Pass on any questions or requests you have for the social hosts.
- Tally and summarize choices.
- Further resources and wrap up.
- How to get involved with next workshops.

**Comment by trallard** [comment]
Logistics: 

- Venue: thinking about Zoom, we can use the Quansight Zoom account
- Event registration: Eventbrite
- Places to advertise: Scientific Python, NumFOCUS newsletter (15th of every month), CZI Science newsletter, social media 
- Recording: yes, preferable (we can put it on the Scientific Python Youtube)
- Can do a write up for the handbook

**Comment by isabela-pf** [comment]
Date: I think I'd rather go earlier to avoid running into the holidays that often make scheduling difficult. That means **Tuesday, September 17** and **Tuesday, October 15** barring other rescheduling needed.

**Comment by isabela-pf** [comment]
Draft event checklist for your review! I want to make sure we're on the same page about what should be done. Please let me know what you think it missing or anything you think might be inaccurate.

## Before event
- Zoom event set up
- Preset Zoom settings
- Event registration set up
- Advertise event via NumFocus
- Advertise event via other mailing lists
- Advertise event via Scientific Python channels
- Advertise via Quansight Labs blog post
- Illustrate for advertisement
- Illustrate for slides
- Make event agenda and notes
- Schedule/arrange captioning
- Schedule event moderators
-  Compile extra resources
- Make feedback survey

## One week before event
- Slides are complete
- Dry run of event transitions and interactions 
- Reminder/join now email for registrees 

## During event
- Team appears early
- Internal resources document
- Test screen share before starting
- Reply to comments in chat where possible
- Refer to agenda document and notes
- Post questions/activities as planned
- Keep time/time check in signal

## After event
- Thank you email
- Captioning and transcription for recording
- Illustrate thumbnail
- Upload recording to YouTube
- Summary and resource blog post to Quansight Labs
- Link recording in other advertising areas as follow up
- Send feedback survey
